_posts/2016-10-24-deploy-jekyll-with-travis.md

```diff
@@ -0,0 +1,232 @@
+---
+layout: post
+title: Deploying Jekyll to Github Pages with Travis and Gulp
+date: 2016-10-24
+author: Anne Tomasevich
+tags: jekyll travis
+summary: |
+  A step-by-step guide to how we deploy our gulpified Jekyll site to GitHub
+  Pages using Travis, opening the door for us to use Jekyll Plugins.
+---
+
+Since our company's inception, we at Savas Labs have used [Jekyll](https://jekyllrb.com/) to build our company website and [GitHub Pages](https://pages.github.com/) to host it with great success. Jekyll is a powerful tool out of the box, and it's hard to imagine a simpler hosting strategy than GitHub Pages, which only requires a [specifically named branch](https://help.github.com/articles/configuring-a-publishing-source-for-github-pages/) and a [single settings update](https://help.github.com/articles/configuring-a-publishing-source-for-github-pages/#enabling-github-pages-to-publish-your-site-from-master-or-gh-pages) to get your site live for free.
```

Comment:
I think you could strengthen the introduction to the post with the why Jekyll works well for us started more clearly. Perhaps link to the [Drupal tag](http://savaslabs.com/blog/tag/drupal/) and say something to the effect, although we often build complex, data heavy systems that need to have a simple workflow for site admins, we recognized there was a tool that was better for our workflow that mitigated security risks, site hosting costs, reduced complexity (without a database back-end) and improved performance